Gemfile.lock

```diff
@@ -0,0 +1,216 @@
+GEM
```

Comment:
- Removed Gemfile.lock
- Added Gemfile.lock to .gitignore
- Rebased and pushed updated branch. You may need to fetch to get the latest branch.